packages/buidler-core/src/internal/buidler-evm/stack-traces/model.ts

```diff
@@ -124,6 +125,7 @@ export class Contract {
 
   private _constructor: ContractFunction | undefined;
   private _fallback: ContractFunction | undefined;
+  private _receive: ContractFunction | undefined;
```

Comment:
@alcuadrado I couldn't think of a better way of doing this that just duplicating the logic around `_fallback`. I think it makes sense since you can have both of them, one of them or neither.

Reply:
Yeah, this makes sense.  While they are similar, we should treat them as separate entities.